segmentstore/server/src/test/java/io/pravega/segmentstore/server/tables/TableContext.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.tables;
+
+import io.pravega.segmentstore.server.CacheManager;
+import io.pravega.segmentstore.server.CachePolicy;
+import io.pravega.segmentstore.server.SegmentContainer;
+import io.pravega.segmentstore.server.UpdateableSegmentMetadata;
+import io.pravega.segmentstore.server.containers.StreamSegmentMetadata;
+import io.pravega.segmentstore.storage.cache.CacheStorage;
+import io.pravega.segmentstore.storage.cache.DirectMemoryCache;
+import java.util.Random;
+import java.util.concurrent.ScheduledExecutorService;
+import lombok.val;
+
+public class TableContext implements AutoCloseable {
```

Comment:
Please add a quick Javadoc on this class explaining its purpose. I know it's in the test package, but since you took it out of the class it was originally in, we need to explain what it should be used for.